### ISSUE TITLE
[FEATURE]Help is needed to fine-tune the cellpose model for higher-quality nanostring data

### ISSUE THREAD

**Issue by yingsun-ucsd** [MouseLand/cellpose#778]
We have some nanostring data for the human pancreas. We want to redo the segments with the images using cellpose for higher quality. I'm very frustrated because I am new to spatial sequencing and have no idea how to start. I did follow squidpy tutorials for nanostring analysis and got some results. However, we had some concerns about it. That is why we want to redo it from the beginning, which is the image (DAPI perhaps). And here the frustration comes because I don't know what I should do.

I hope to get some help and suggestions here, such as:
1) What images I should use? Nanostring has a whole bunch of JPG and TIF images: cell composite, cell labels, cell overlay, and compartment labels (those four groups are what we have). Do I need to get any additional images from nano string?
2) a protocol I can follow with some examples if possible.
3) any other suggestions to solve the problems we have

Thank you very much!

**Comment by natelharrison** [comment]
Hi, sorry I do not have a lot of biology experience nor am I familiar with Nanostring (I just work with the data I am given) but I can share some of my experience training on zebrafish cells. I'm not sure what your images are like so I am just assuming they are whole cells you want to segment.

1) Cellpose takes in 2D images composed of image and image mask pairs. So you would have image_1.tif and image_1_masks.tif in a single directory. You can segment whole cells and nuclei, though I only have experience with cell segmentation so I'm not sure how you would structure the training data for both. You can do either TIF or JPG, whichever you prefer though I recommend TIF.

2) This depends on whether you want to train through the GUI or CLI. They have a [youtube tutorial](https://www.youtube.com/watch?v=5qANHWoubZU&pp=ygUMY2VsbHBvc2UgMi4w) for GUI that will also allow you to use their human-in-the-loop training, which is useful if your dataset is rather small. If you have a decent amount of data you can also fine-tune one of their models through the CLI by following this [guide](https://cellpose.readthedocs.io/en/latest/train.html).

3. A few tips are to make sure that the background for the image mask has a value of 0 and to make sure that the labels are as high quality as can be. If you have any areas that aren't labeled accurately exclude them entirely from the training data or manually correct those areas.

**Comment by yingsun-ucsd** [comment]
@natelharrison Thank you very much for your help. It's very helpful. I will look at it in detail and try on my datasets. I will reach out here for further questions I might have later.

**Comment by mrariden** [comment]
closing for now
